projects/adin1300_dual/a10soc/system_project.tcl

```diff
@@ -0,0 +1,131 @@
+# Triyng to use other pins, this issue migth not be true in the future
+###############################################################################
+# Note: This projects requires a hardware rework to function correctly.
+# The rework connects FMC header pins directly to the FPGA so that they can be
+# accessed by the fabric.
+#
+# Changes required:
+#  R610: DNI -> R0
+#  R611: DNI -> R0
+#  R612: R0 -> DNI
+#  R613: R0 -> DNI
+#  R620: DNI -> R0
+#  R632: DNI -> R0
+#  R621: R0 -> DNI
+#  R633: R0 -> DNI
+###############################################################################
+
+
+set_location_assignment PIN_E5     -to ref_clk_125               ; ## H04  FMCA_HPC_CLK0_M2C_P
+set_location_assignment PIN_F5     -to "ref_clk_125(n)"          ; ## H05  FMCA_HPC_CLK0_M2C_N
+
+set_instance_assignment -name IO_STANDARD LVDS -to ref_clk_125
+set_instance_assignment -name IO_STANDARD LVDS -to "ref_clk_125(n)"
+
+set_location_assignment PIN_G14    -to  rgmii_rxc_a               ; ## G06 FMCA_HPC_LA00_CC_P
+set_location_assignment PIN_B9     -to  rgmii_rx_ctl_a            ; ## H14 FMCA_HPC_LA07_N
+set_location_assignment PIN_C13    -to  rgmii_rxd_a[0]            ; ## H07 FMCA_HPC_LA02_P
+set_location_assignment PIN_D13    -to  rgmii_rxd_a[1]            ; ## H08 FMCA_HPC_LA02_N
+set_location_assignment PIN_C14    -to  rgmii_rxd_a[2]            ; ## G09 FMCA_HPC_LA03_P
+set_location_assignment PIN_D14    -to  rgmii_rxd_a[3]            ; ## G10 FMCA_HPC_LA03_N
+set_location_assignment PIN_H13    -to  rgmii_txc_a               ; ## H11 FMCA_HPC_LA04_N
+set_location_assignment PIN_A9     -to  rgmii_tx_ctl_a            ; ## H13 FMCA_HPC_LA07_P
+set_location_assignment PIN_A12    -to  rgmii_txd_a[0]            ; ## D14 FMCA_HPC_LA09_P
+set_location_assignment PIN_A13    -to  rgmii_txd_a[1]            ; ## D15 FMCA_HPC_LA09_N
+set_location_assignment PIN_A10    -to  rgmii_txd_a[2]            ; ## C10 FMCA_HPC_LA06_P
+set_location_assignment PIN_B10    -to  rgmii_txd_a[3]            ; ## C11 FMCA_HPC_LA06_N
+
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_rxc_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_rx_ctl_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_rxd_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_txc_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_tx_ctl_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to rgmii_txd_a
+
+set_location_assignment PIN_C9     -to  mdio_fmc_a                ; ## H16 FMCA_HPC_LA11_P
+set_location_assignment PIN_D9     -to  mdc_fmc_a                 ; ## H17 FMCA_HPC_LA11_N
+
+set_instance_assignment -name IO_STANDARD "1.8 V" -to mdio_fmc_a
+set_instance_assignment -name WEAK_PULL_UP_RESISTOR ON -to mdio_fmc_a
+set_instance_assignment -name IO_STANDARD "1.8 V" -to mdc_fmc_a
+
+set_location_assignment PIN_E12    -to  gp_clk_a                  ; ## D08 FMCA_HPC_LA01_CC_P - not used in RGMII mode
```

Comment:
This pin is not used in the design. Either to be removed from system project or added to system top. This pin will need rework of the board, but given that it's not used in the design I would remove it from the A10SOC project altogether.

Reply:
Done.